connectors/cli.py

```diff
@@ -140,10 +141,16 @@ def run(args):
     coro = _start_service(config, args, loop)
 
     try:
+        Connector.es_index = ConnectorIndex(config["elasticsearch"])
```

Comment:
There's a small design issue. You want to create your indexes and pass them to the Services you create, and deal with their shutdown there.  The `cli` module should only deal with `Service` classes.